packages/hplip.rb

```diff
@@ -3,17 +3,24 @@
 class Hplip < Package
   description 'Print, Scan and Fax Drivers for Linux'
   homepage 'https://developers.hp.com/hp-linux-imaging-and-printing/'
-  version '3.18.7'
-  source_url 'https://sourceforge.net/projects/hplip/files/hplip/3.18.7/hplip-3.18.7.tar.gz'
-  source_sha256 '55b879e1d6d0d88c32d79486b748a21759cab404036d883be4ea9fcb55bfeb1d'
+  version '3.20.2'
+  source_url 'https://sourceforge.net/projects/hplip/files/hplip/3.20.2/hplip-3.20.2.tar.gz'
+  source_sha256 '90c49d74688b4d745a739a6db9bf8dbdfa134c24e921e31909bffe9d84f471c2'
 
   binary_url ({
   })
   binary_sha256 ({
   })
 
-  depends_on 'libjpeg'
   depends_on 'cups'
+  depends_on 'dbus'
+  depends_on 'libjpeg'
```

Comment:
`depends_on 'libjpeg_turbo'`?

Reply:
We should probably do a bulk update for that in it's own PR. I'll update this and then do a bulk update.